### DISCUSSION TITLE
Multiple functions app in the same repo

### DISCUSSION THREAD

**Discussion by martinGM05** [Azure/Azure-Functions#2502] [Polls]
I need include multiple functions app in the same repo, like a stacks

src/functions/payers
src/functions/providers

I have my repo in Typescript

**Comment by juanchomanza** [comment]
To include multiple Azure Functions apps in the same TypeScript monorepo (like a stack with src/functions/payers, src/functions/providers, etc.), you can absolutely do that by organizing your code properly and configuring each app with its own function.json, host.json, and tsconfig.json if needed.

Like this:

src/
└── functions/
    ├── payers/
    │   ├── index.ts
    │   └── function.json
    └── providers/
        ├── index.ts
        └── function.json
